test/browser/render.test.js

```diff
@@ -241,6 +241,14 @@ describe('render()', () => {
 		expect(root.children[3]).to.have.property('value', '');
 	});
 
+	it('should set value inside the specified range', () => {
+		render(
+			<input type="range" value={0.5} min="0" max="1" step="0.05" />,
+			scratch
+		);
```

Comment:
_(Note for my understanding)_ - So in this test case, if the `type` property is set to `range` and then `value` is set before `min`, `max` or `step` then `value` will be rounded to "1" because the [default values for min, max and step](https://html.spec.whatwg.org/multipage/input.html#range-state-(type=range)) are 0, 100 and 1 respectively.

Reply:
That's exactly what happens 👍 The browser won't update the value when the range boundaries change.